dbrequests/mysql/tests/test_send_delete.py

```diff
@@ -0,0 +1,108 @@
+"""Testing send_data functionality."""
+
+import datatable as dt
+import pytest
+from dbrequests.mysql.tests.conftest import (
+    set_up_diffs as reset_diffs)
+
+
+@pytest.mark.usefixtures('db')
+class TestSendDeleteInSet:
+    """Tests for mode=in_set."""
+
+    def test_delete_entries(self, db):
+        """Delete some entries from a table."""
+        reset_diffs(db)
+        df = dt.Frame({
+            'id': [1, 2, 3],
+            'value': ['a', 'b', 'c']
+        })
+
+        df_delete = dt.Frame({
+            # ids we want to delete:
+            'id': [1, 2]
+        })
+
+        db.send_data(df, 'diffs')
+        nrows = db.send_delete(df_delete, 'diffs', 'in_set')
+        res = db.send_query('select * from diffs', to_pandas=False)
+        assert nrows == 2
+        assert res.shape == (1, 3)
+        assert res[0, 'id'] == 3
```

Comment:
Zu den beiden set-bezogenen Methoden würde ich mir noch Tests wünschen, die nicht nur auf eine, sondern auf zwei Spalten gehen. Die "and"-Verknüpfung bedeutet ja "gefühlt" auch etwas anderes, wenn man mit "in" vs. "not in" arbeitet.

Reply:
Ist umgesetzt in https://github.com/INWTlab/dbrequests/pull/39/commits/0acd6a900817865ce15d3b7e5fd1dc7e6e463a55